.github/workflows/build.yml

```diff
@@ -76,6 +88,7 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: build_${{ matrix.os.runner }}_${{ matrix.os.cxx }}
+          name: build_${{ matrix.os.runner }}${{ matrix.os.preset == 'linux-32' && '-32' || '' }}_${{ matrix.os.cxx }}
```

Comment:
You can change it with `name: build_${{ matrix.os.preset }}_${{ matrix.os.cxx }}`.